x-pack/plugins/ml/server/models/data_frame_analytics/validation.ts

```diff
@@ -236,6 +237,7 @@ async function getValidationCheckMessages(
       size: 0,
       track_total_hits: true,
       body: {
+        ...(source.runtime_mappings ? { runtime_mappings: source.runtime_mappings } : {}),
```

Comment:
I think we can use `isRuntimeMappings` here

Reply:
I don't think there's a need to here as that check is done before the config is sent to this endpoint - so we know if `runtime_mappings` is defined that it it passed `isRuntimeMappings` vetting 👍